s2i-java-11/s2i-java-11-task.yaml

```diff
@@ -35,14 +40,19 @@ spec:
         - '/bin/sh'
         - '-c'
       args:
-        - echo "MAVEN_CLEAR_REPO=${inputs.params.MAVEN_CLEAR_REPO}" > env-file &&
-          if [[ ${inputs.params.MAVEN_ARGS_APPEND} != "skip-if-not-set" ]]; then
-            echo "MAVEN_ARGS_APPEND=${inputs.params.MAVEN_ARGS_APPEND}" >> env-file;
-          fi &&
-          if [[ ${inputs.params.MAVEN_MIRROR_URL} != "skip-if-not-set" ]]; then
-            echo "MAVEN_MIRROR_URL=${inputs.params.MAVEN_MIRROR_URL}" >> env-file;
-          fi &&
+        - |-
+          echo "MAVEN_CLEAR_REPO=$(inputs.params.MAVEN_CLEAR_REPO)" > env-file
+
+          [[ '$(inputs.params.MAVEN_ARGS_APPEND)' != "" ]] &&
```

Comment:
how about 
[[ -n inputs.params.MAVEN_ARGS_APPEND ]]

Reply:
@nikhil-thomas may be possible to use [[ -n '$(inputs.param.)' ]] (note the quote `'`) but I haven't tested it ; since the one in PR above works I don't see why `-n ` wouldn't

Reply:
current state is good enough